releasetasks/templates/fennec/candidates_fennec.yml

```diff
@@ -0,0 +1,57 @@
+-
+    taskId: "{{ stableSlugId(buildername) }}"
+    reruns: 5
+    task:
+        provisionerId: aws-provisioner-v1
+        workerType: gecko-decision
+        created: "{{ now }}"
+        deadline: "{{ now.replace(days=4) }}"
+        expires: "{{ never }}"
+        priority: "high"
+        retries: 5
+        routes: []
+        scopes:
+             - docker-worker:cache:level-3-*
+             - docker-worker:cache:tooltool-cache
+             - scheduler:extend-task-graph:*
+             - assume:hook-id:project-releng/candidates-fennec-dev,
+             - queue:create-task:scriptworker-prov-v1/beetmoverworker-v1
+             - queue:create-task:scriptworker-prov-v1/balrogworker-v1
+        payload:
+            maxRunTime: 1800
+            env:
+                GECKO_BASE_REPOSITORY: "https://hg.mozilla.org/{{ repo_path }}"
+                GECKO_HEAD_REPOSITORY: "https://hg.mozilla.org/{{ repo_path }}"
+                GECKO_HEAD_REF: "{{ revision }}"
+                GECKO_HEAD_REV: "{{ revision }}"
+                BUILD_NUMBER: "{{ buildNumber }}"
+            image: taskcluster/decision:0.1.5
+            cache:
+               level-3-hg-shared: /home/worker/hg-shared
+               level-3-checkouts: /home/worker/checkouts
+            features:
+               chainOfTrust: true
+               taskclusterProxy: true
+            command:
+                - /home/worker/bin/run-task
+                - --vcs-checkout=/home/worker/checkouts/gecko
+                - /bin/bash
+                - -cx
+                - >
+                cd /home/worker/checkouts/gecko && ln -s /home/worker/artifacts artifacts && ./mach --log-no-times taskgraph decision --pushlog-id='0' --pushdate='0' --project='jamun' --message='try: -b o -p foo -u none -t none' --owner='asasaki@mozilla.com' --level='3' --base-repository=$GECKO_BASE_REPOSITORY --head-repository=$GECKO_HEAD_REPOSITORY --head-ref=$GECKO_HEAD_REF --head-rev=$GECKO_HEAD_REV --triggered-by='nightly' --target-tasks-method='candidates_fennec'\n
```

Comment:
We want the `--project='jamun'` part to be `--project=mozilla-beta` on m-b and `--project=mozilla-release` on m-r.  Hopefully we have a template string for this?